src/main/java/seedu/duke/quote/QuoteList.java

```diff
@@ -1,7 +1,49 @@
 package seedu.duke.quote;
 
+import seedu.duke.lists.QuotesifyList;
+
 import java.util.ArrayList;
 
-public class QuoteList {
-    private ArrayList<Quote> quotes;
+public class QuoteList extends QuotesifyList<Quote> {
+    private ArrayList<Quote> quotes = super.getList();
+
+    public QuoteList() {
+        super(new ArrayList<>());
+    }
+
+    public QuoteList(ArrayList<Quote> quotes) {
+        super(quotes);
+    }
+
+    @Override
+    public void add(Quote newQuote) {
+        quotes.add(newQuote);
+    }
+
+    @Override
+    public void delete(int index) {
+        quotes.remove(index);
+    }
+
+    @Override
+    public String toString() {
+        return "";
+    }
+
+    public void printAllQuotes() {
+        System.out.println("Here are your quotes:");
+        for (Quote quote : quotes) {
+            String quoteToPrint = '\"' + quote.getQuote() + '\"';
```

Comment:
I was thinking if it would be better to place quoteToPrint inside your toString() method and implement printAllQuotes() inside TextUi class instead? Then all you need is to print quoteList.toString() inside your printAllQuotes() method to print all quotes. 

What do you think? :)

Reply:
That's a good suggestion. I have updated the code to reflect the changes. Now the printing of quotes will be done with the TextUI class!